Makefile

```diff
@@ -57,3 +57,8 @@ srpm:
 
 distclean: clean
 	rm -rf $(BUILDDIR)
+
+debug:
+	$(MAKE) clean
+	@sed -i 's/debug=false/debug=true/g' meson.build
+	$(MAKE) build
```

Comment:
Please avoid running sed on the files checked into git -- just pass the `-Ddebug=true` option to `meson setup`, perhaps by making the `make build` rule utilize a `$(MESONARGS)` variable or something and setting that here.

Reply:
I agree. Also, wouldn't it be sufficient to use `meson configure`? For example:
```bash
# disable debug
meson configure builddir -Ddebug=false
# enable debug
meson configure builddir -Ddebug=true
```
This should let you switch without the need to re-setup the project. For the make users, we can also add a new target for it.